network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -152,7 +153,8 @@ class BuyersGuideProductCategory(index.Indexed, TranslatableMixin, LocalizedSnip
 
     slug = models.SlugField(
         blank=True,
-        help_text='A URL-friendly version of the category name. This is an auto-generated field.'
+        help_text='A URL-friendly version of the category name. This is an auto-generated field.',
+        max_length=100
```

Comment:
Hi @TheoChevalier! It turns out the error was being caused by the slug field, according to the django docs, the default max length for a slug field is 50! Updating this and migrating on my local machine solved the issue

Reply:
Thanks Daniel, nice find :)